honeybee_radiance_command/options/rcontrib.py

```diff
@@ -249,3 +249,26 @@ def t(self):
     @t.setter
     def t(self, value):
         self._t.value = value
+
+    def to_radiance(self):
+        """Translate options to Radiance format."""
+
+        # Note on overriding this method here. (@sariths 29.09.2021)
+        # The options below have an impact of how modifiers specified through m and M will behave.
+        # So, I am checking for these upfront and placing them in the beginning of the options.
+        positional_options = ('_p', '_b', '_bn', '_o')
+
+        options = ""
+        for option_flag in positional_options:
+            if option_flag in self.slots:
```

Comment:
Do we really need this check? These options will always be in slots. If it is necessary, then because the `self.slots` method [runs every single time](https://github.com/ladybug-tools/honeybee-radiance-command/blob/1a3b7d3247ac3f5fd3ddb494d4d6dfb3ee0c4a73/honeybee_radiance_command/options/optionbase.py#L408) to get the slots I would call it one and then reuse it here.

```python
       slots = self.slots
       
       for option_flag in positional_options:
            if option_flag in slots:
```